e2e/runtime_test.go

```
@@ -204,17 +204,17 @@ func TestRuntime(t *testing.T) {
 				containerName)
 			require.NoError(t, err, "stderr: %s", stderr)
 			t.Cleanup(func() {
-				_, _, _ = installedRuntimeSudo.Run("delete", containerName)
```

Just to follow up on my concerns from yesterday, I think what you've done is the correct behaviour for podman. Since the root environment and the user environment are different. 

Although.. I'm a bit confused why this test worked in the first place.

My understanding of podman is that root and user environments are completely separate, in which case your changes are fine. But the fact this test worked in the first place confuses me a bit. If it's just that the delete was silently failing.. then that's fine. But if it's thy the sudo delete was working- maybe we should try to preserve Podmans default behaviour?